tests/pos-java-interop/i2760/Test.scala

```diff
@@ -0,0 +1 @@
+@Fork(value = 16) class Test
```

Comment:
How do we know we get the right default in the backend? Can we make this a run test?

Reply:
Yes, done :).